hack/lib/golang.sh

```diff
@@ -24,6 +24,7 @@ readonly KUBE_GOPATH="${KUBE_OUTPUT}/go"
 # If you update this list, please also update build/release-tars/BUILD.
 kube::golang::server_targets() {
   local targets=(
+    cmd/gke-certificates-controller
```

Comment:
that would probably work! I'm not sure whether we want to make another grouping for this (I don't know what to call it) or just add to the list directly.